src/styles.css

```diff
@@ -0,0 +1,35 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+
+.App {
+  font-family: Roboto, Arial, sans-serif;
+  text-align: center;
+  color: white;
+}
+
+body {
+  background: #181818;
+  margin: 0;
+}
+
+.body-container {
+  margin-top: 3.2rem;
+  margin-left: 4rem;
+  margin-right: 4rem;
+}
+
+.section-heading {
+  font-weight: 500;
+  padding-top: 1rem;
+}
+
+.Toastify__progress-bar {
```

Comment:
Please take a look at this classname, you can keep it consistent with rest of your css classes.

Reply:
This classname is from an external library component, I'm overriding it to my use case

Reply:
sure.